departments.html

```diff
@@ -0,0 +1,586 @@
+<!DOCTYPE html>
+<html lang="ru">
+
+<head>
+  <title>СУНЦ УрФУ | Официальный сайт</title>
+  <meta content="text/html; charset=utf-8" http-equiv="Content-Type">
+  <meta content="width=device-width, initial-scale=1" name="viewport">
+  <link href="assets/img/favicon.ico" rel="icon" type="image/x-icon">
+  <link href="//fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
+  <link href="assets/css/spectre.css" rel="stylesheet" type="text/css">
+  <link href="assets/css/site-section.css" rel="stylesheet" type="text/css">
+  <link href="assets/css/study.css" rel="stylesheet" type="text/css">
+  <link href="assets/css/icons.css" rel="stylesheet" type="text/css">
+</head>
+
+<body>
+  <header>
+    <section class="grid-header container grid-960 pt-10 pb-10">
+      <nav class="navbar">
+        <section class="nav-left navbar-section">
+          <a class="no-outline" href="index.html"><img alt="СУНЦ УрФУ" class="img-responsive" id="logo" src="assets/img/logo-no-text.png"></a>
+          <div class="hide-md">
+            <a class="btn btn-link" href="about.html">О нас</a>
```

Comment:
The comments here should start at `</a>` and end at the next `<a>`. This needs to be fixed in all files that contain the header